src/main/java/edu/cornell/kfs/module/purap/batch/JaggaerGenerateSupplierXmlStep.java

```diff
@@ -0,0 +1,96 @@
+package edu.cornell.kfs.module.purap.batch;
+
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+import org.kuali.kfs.coreservice.impl.parameter.Parameter;
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.module.purap.CUPurapParameterConstants;
+import edu.cornell.kfs.module.purap.CUPurapConstants.JaggaerContractUploadProcessingMode;
+import edu.cornell.kfs.module.purap.batch.service.JaggaerGenerateContractPartyCsvService;
+import edu.cornell.kfs.module.purap.batch.service.JaggaerGenerateSupplierXmlService;
+import edu.cornell.kfs.module.purap.jaggaer.supplier.xml.SupplierSyncMessage;
+import edu.cornell.kfs.sys.CUKFSConstants;
+
+public class JaggaerGenerateSupplierXmlStep extends AbstractStep {
+    private static final Logger LOG = LogManager.getLogger();
+    
+    protected JaggaerGenerateSupplierXmlService jaggaerGenerateSupplierXmlService;
+    protected ParameterService parameterService;
+    
+    @Override
+    public boolean execute(String jobName, java.util.Date jobRunDate) throws InterruptedException {
+        JaggaerContractUploadProcessingMode processingMode = findJaggaerContractUploadProcessingMode();
```

Comment:
I think I am going to create a new enum JaggaerUploadSuppliersProcessingMode rather than re-use JaggaerContractUploadProcessingMode.  I expect JaggaerContractUploadProcessingMode will be removed when we are ready to remove jaggaerGenerateContractPartyCsvJob.  This approach will more cleanly separate this new batch job from the other.